src/time/Storage.cpp

```diff
@@ -92,6 +98,15 @@ Eigen::VectorXd Storage::getValuesAtOrAfter(double before) const
   return stample->sample.values;
 }
 
+//@todo merge getValuesAtOrAfter and getGradientsAtOrAfter into getSampleAtOrAfter, then let user draw data from Sample
```

Comment:
This requires also some changes in the tests, but makes sense, if we don't want to always have to do some specific implementation just for gradients.